src/Mailer/Email.php

```diff
@@ -1830,6 +1841,29 @@ public function message($type = null)
     }
 
     /**
+     * Sets priority.
+     *
+     * @param int|null $priority 1 (highest) to 5 (lowest)
+     * @return $this
+     */
+    public function setPriority($priority)
+    {
+        $this->_priority = $priority;
```

Comment:
> So far we never do too much validation here for those which are documented clearly enough.

I feel that is a mistake. I feel like the value of having all these split methods is to help catch and prevent errors earlier. Without that, we've made very fancy public properties.